### PR TITLE
Make JSON Schema and example blocks collapsable

### PR DESCRIPTION
**Pull request by EpocDotFr**
Because they are often big.

image

### CODE REVIEW

File: lib/template.nunjucks [hunk 1]
```diff
@@ -47,6 +47,24 @@
             }
           } catch(e) {}
         });
+
+        // allow to show/hide JSON shemas or examples
+        $('.toggle-schema-or-example').on('click', function(e) {
+            e.preventDefault();
```
Comment (kevinrenskers) [inline lib/template.nunjucks]: Can you indent with 2 spaces please? Thanks :)

Comment (EpocDotFr) [inline lib/template.nunjucks]: Whoops, sure!

**Comment by kevinrenskers** [comment]
I tried it with the example RAML file and I have to say that I am not a huge fan of this, at least not when the default is for the blocks to be hidden.

screen shot 2016-08-10 at 11 20 30

This example block is small and crucial. It's *the* documentation on how to use this endpoint, but it requires an extra click to show it.

Maybe it would be better to only do it for JSON Schemes, not for examples?

**Comment by kevinrenskers** [comment]
Also wondering if this makes sense for the default template that ships with raml2html. After all, it's pretty easy to use your own customized templates as you've just showed :)

**Comment by EpocDotFr** [comment]
I'm working on a pretty big API. Showing all JSON Schema and examples (that are pretty big too) put our mobile developers angry because the endpoint docs was hard to read. That's why I cloned the default template ^^ And I was wondering "hey, maybe it would be cool to share my modifications in the default template!".

But you're probably right :)

**Comment by kevinrenskers** [comment]
No, then we end up with 100 switches for every feature someone wants :) It's also one more thing that we are then supposed to maintain going forward, one more thing locked in the CLI. With brand new templates coming up for version 4 of raml2html I also think that this is not the right time to add this feature in the old templates, sorry.

Let's ship raml2html with one default template without switches and toggles. The goal is to be a simple RAML documentation renderer. And because we know that it won't be perfect for everyone, templates can be customized.

Thanks for the PR though, and by all means please keep contributing! :)

**Pull request closed without merging**